config/spring-security-config.gradle

```diff
@@ -130,3 +129,33 @@ tasks.withType(KotlinCompile).configureEach {
 }
 
 build.dependsOn rncToXsd
+
+compileTestJava {
```

Comment:
Is it true that we don't want to run these tests against JDK 8? It seems to me that we want to run the tests twice: Once for JDK 8 and once for JDK 11.

Reply:
We want to run they using JDK 11 only because they are using OpenSAML4 classes